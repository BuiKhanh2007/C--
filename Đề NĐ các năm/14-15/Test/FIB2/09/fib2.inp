100
10 80
29 800401
5 5
3 2
1 1
11 47
27 86352
14 283
24 16107
3 1
0 1
10 87
23 13220
19 1841
24 23991
7 21
27 307777
32 2127762
31 1683788
6 13
19 3241
9 52
14 508
30 127078
19 2963
6 11
29 747663
31 707795
3 3
32 2677785
25 120558
11 29
1 1
12 73
2 2
29 498135
3 2
9 42
22 19578
16 1471
27 92706
20 5775
28 3880
2 1
8 10
10 8
8 13
28 125770
28 155485
5 1
9 20
12 228
21 3317
16 516
21 2784
28 395831
21 643
4 4
8 18
3 3
27 116473
11 133
18 3
8 2
9 11
18 2344
17 941
26 84037
10 12
1 1
9 40
32 1645002
14 473
11 66
21 10788
7 3
9 52
9 25
12 188
19 6043
17 1367
22 7932
5 8
14 499
7 15
23 12177
29 175363
21 10112
32 1746653
17 1252
31 1971425
5 1
15 315
22 22828
28 344115
30 81063
20 1278
2 1
18 1667
26 86510
