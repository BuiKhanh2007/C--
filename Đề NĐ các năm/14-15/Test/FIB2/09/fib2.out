31
305726
2
1
0
18
32984
108
6152
0
1
33
5050
703
9164
8
117560
812733
643150
5
1238
20
194
48540
1132
4
285582
270354
1
1022823
46049
11
0
28
1
190271
1
16
7478
562
35411
2206
1482
1
4
3
5
48040
59390
0
8
87
1267
197
1064
151194
246
2
7
1
44489
51
1
1
4
895
360
32099
5
0
15
628335
181
25
4121
1
20
10
72
2308
522
3030
3
191
6
4651
66983
3863
667162
478
753017
0
120
8720
131440
30963
488
1
637
33044
